vms/evm/database/blockdb/database.go

```diff
@@ -0,0 +1,528 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package blockdb
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"path/filepath"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/ethdb"
+	"github.com/ava-labs/libevm/rlp"
+	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/zap"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/database/heightindexdb/meterdb"
+	"github.com/ava-labs/avalanchego/database/prefixdb"
+	"github.com/ava-labs/avalanchego/utils/logging"
+
+	heightindexdb "github.com/ava-labs/avalanchego/x/blockdb"
+)
+
+var (
```

Comment:
Although it's idiomatic in other languages, it's not a requirement in Go to declare all globals at the top like this. The primary drawback is that they lack context relative to where they're used so a reader has to jump back and forth between declaration and usage sites.

Please move the declarations (perhaps still in groups, but also feel free to break them up if warranted) closer to where they're used.